maven-scm-providers/maven-scm-providers-git/maven-scm-provider-gitexe/src/test/java/org/apache/maven/scm/provider/git/gitexe/command/status/GitStatusConsumerTest.java

```diff
@@ -52,7 +53,8 @@
     {
         GitStatusConsumer consumer = new GitStatusConsumer( new DefaultLog(), null, relativeRepoPath );
 
-        try ( BufferedReader r = new BufferedReader( new FileReader( gitlog ) ) )
+        try ( BufferedReader r = new BufferedReader(
+                new InputStreamReader ( new FileInputStream( gitlog ), StandardCharsets.UTF_8 ) ) )
```

Comment:
I don't know whether this is better. If we read from stdin or the stdin has been redirected and the encoding was not UTF-8 and did contain multibyte characters the reader will show errors.